EnableJupyterDisplay.swift

```diff
@@ -0,0 +1,117 @@
+import Cryptor
```

Comment:
I just had an idea. You could do `#if canImport(Cryptor)` around all the Cryptor stuff, so that this still works (just without message signing) if the user hasn't installed Cryptor.

You could also have an `#if not canImport(Cryptor)` that checks what the key is and emits an error if the user tries to include this without Cryptor if there is a key.